recipes/python-abi3/bld.bat

```diff
@@ -0,0 +1,11 @@
+:: Copy the [de]activate scripts to %PREFIX%\etc\conda\[de]activate.d.
+:: This will allow them to be run on environment activation.
+FOR %%F IN (activate deactivate) DO (
+    if not exist %PREFIX%\etc\conda\%%F.d MKDIR %PREFIX%\etc\conda\%%F.d
+    if errorlevel 1 exit 1
+    sed "s/@PY_LIMITED_API/%PY_LIMITED_API%/g" %RECIPE_DIR%\scripts\%%F.bat >> %PREFIX%\etc\conda\%%F.d\%PKG_NAME%_%%F.bat
```

Comment:
I think you forgot to create the bat versions of the activation scripts. This didn't fail here because this call didn't have an `if errorlevel`.